src/main/java/duke/Controller/Parser.java

```diff
@@ -0,0 +1,78 @@
+package duke.Controller;
```

Comment:
Names representing packages should be in all lower case. Refer to coding standard